frontend/src/pages/admin/Template/sections/Editor.vue

```diff
@@ -49,19 +57,27 @@
 </template>
 
 <script>
-
-import FormRow from '@/components/FormRow'
+import FormRow from '@/components/ForumRow'
```

Comment:
You have a typo here, should be `FormRow`, not `ForumRow`. Please make sure that you're checking the UI runs/works before committing the changes.